deployments/goerli/ERC1155_PREDICATE.json

```diff
@@ -1,97 +1,4 @@
 {
```

Comment:
Why did you change this file ?

Reply:
We should replace it with the actual predicate on the L1 networks, it was deployed because of a bad deploy config (should have been skipped or deployment configured manually like CHILD_CHAIN_MANAGER)